mpqp/core/instruction/measurement/measure.py

```diff
@@ -42,14 +42,16 @@ class Measure(Instruction, ABC):
 
     def __init__(
         self,
-        targets: list[int],
+        targets: Optional[list[int]] = None,
         shots: int = 0,
```

Comment:
Shouldn't we put `shots: int = 1024` by default in `Measure` now that it is the case for both `BasisMeasure` and `ExpectationMeasure` ?